public/data/deborshikun.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Deborshi Chakrabarti",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/deborshikun.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub!!!",
+      "url": "https://github.com/deborshikun",
+      "icon": "github"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Learning about Open Source Contributions",
+      "date": "July 2022",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Started Open Source learning"
+    }
+  ]
+}
+
```

Comment:
There seems to be an extra line here. GitHub doesn't display the endline.
```suggestion
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant

Reply:
Yup I checked that too! seemed like I missed it on the vim. I have applied the necessary changes!